docs/overview/getting-started/intro.md

```diff
@@ -2,69 +2,53 @@
+interoperable, highly scalable ecosystem. Avalanche is the first ecosystem designed to accommodate
+the scale of global finance, with near-instant transaction finality.
 
-[Avalanche](https://avax.network) is an open-source platform for launching [decentralized
-applications](https://support.avalabs.org/en/articles/4587146-what-is-a-decentralized-application-dapp)
-and enterprise [blockchain](http://support.avalabs.org/en/articles/4064677-what-is-a-blockchain)
-deployments in one interoperable, highly scalable ecosystem. Avalanche is the first decentralized
-smart contracts platform built for the scale of global finance, with near-instant transaction
-finality. Ethereum developers can quickly build on Avalanche as Solidity works out-of-the-box.
+Avalanche empowers developers to join a rich ecosystem of blockchains known as Subnets. Subnets
+are isolated clusters of the Avalanche network that allow any Dapp to run on its own dedicated set
+of nodes, enabling unmatched levels of fault-tolerance and customization for any use-case. Each
+Subnet is isolated from other Subnets in the ecosystem, so increased usage on a single Subnet 
+doesn't affect other Subnets on Avalanche.
 
-A key difference between Avalanche and other decentralized networks is the consensus protocol. Over
-time, people have come to a false understanding that blockchains have to be slow and not scalable.
-The Avalanche protocol employs a novel approach to consensus to achieve its strong safety
-guarantees, quick finality, and high-throughput without compromising decentralization.
+In Avalanche, developers can create their Dapps by implementing them as a Virtual Machine (VM).
+EVM developers can build on Avalanche's implementation of the EVM straight out-of-the box, or build
+their own custom VM for advanced use cases. VMs are supported in multiple languages including Go
+and Rust, giving developers unparalleled amounts of flexibility in their implementation.
 
-## AVAX
+Avalanche employs a novel proof-of-stake consensus algorithm which offers strong safety guarantees,
+quick finality times, and high throughput, without compromising decentralization. Avalanche is
+incredibly energy-efficient and sustainable and can run on easily accessible consumer-grade
+hardware, consuming the same amount of energy as 46 US households. The entire Avalanche network
+consumes only 0.0005% of the amount of energy consumed by Bitcoin.
 
-AVAX is the native token of Avalanche. It’s a hard-capped, scarce asset that is used to pay for
-fees, secure the platform through staking, and provide a basic unit of account between the multiple
-Subnets created on Avalanche. `1 nAVAX` is equal to `0.000000001 AVAX`.
+## Building on Avalanche
 
-## Avalanche Consensus Protocol
+### Virtual Machines
 
-![Consensus Comparison](/img/Consensus-protocol-comparison.png)
+Avalanche allows developers to build Dapps through the Virtual Machine (VM) framework. VMs are 
```

Comment:
Yeah, I don't like this `through the Virtual Machine framework` framing.

Reply:
Yeah i think this might seem confusing; reworded to make it clear that you don't have to make your own vm necessarily